frontend/src/reducers/productReducers.js

```diff
@@ -35,6 +38,22 @@ export const productListReducer = (
   }
 };
 
+export const productCategoryListReducer = (
+  state = { loading: true, products: [] },
```

Comment:
Why on this line was 'products' not switched to 'categories' ?

Or is it simply not being utilized at all and has nothing to do with the SUCCESS categories return, and therefore could have been deleted?

Reply:
Most likely due to the fact that we're returning distinct products based on the category property of Product. On the models there is no dedicated Category model therefore no category._id hence products cannot be switched to categories.